api4/system_test.go

```diff
@@ -208,6 +208,23 @@ func TestGenerateSupportPacket(t *testing.T) {
 		require.NotZero(t, len(file))
 	})
 
+	t.Run("As a System Administrator but with RestrictSystemAdmin true", func(t *testing.T) {
+		l := model.NewTestLicense()
```

Comment:
Is this needed ?

Reply:
Not really because on the endpoint, the line that checks for permissions comes before the check for the license. But no harm in setting up the test with a license

Reply:
If it's not relevant for the test we should remove it to keep the code cleaner

Reply:
Removed it

Reply:
Thanks!